### PR TITLE
change url of library request

### PR DESCRIPTION
**Pull request by joshua-8**
I want to change the url from RCMgames/BSCD to RCMgames/BSED. Thank you!

### CODE REVIEW

**Review by per1234 — COMMENTED** [review]
Hello @joshua-8. I am confirming your request.

Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests. They will merge this pull request when it is done. You will get a notification from GitHub at that time.

**Review by MatteoPologruto — APPROVED** [review]
Done:
```
2024/05/13 14:17:58 Loaded 6904 libraries from DB
Changing URL of library byte-sized-encoder-decoder from https://github.com/RCMgames/BSCD.git to https://github.com/RCMgames/BSED.git
Success!
```

**Pull request merged**
